lib/libzfs/libzfs_util.c

```diff
@@ -327,7 +327,8 @@ zfs_verror(libzfs_handle_t *hdl, int error, const char *fmt, va_list ap)
 	if (hdl->libzfs_printerr) {
 		if (error == EZFS_UNKNOWN) {
 			(void) fprintf(stderr, dgettext(TEXT_DOMAIN, "internal "
-			    "error: %s\n"), libzfs_error_description(hdl));
+			    "error: %s: %s\n"), libzfs_error_description(hdl),
```

Comment:
This could match the order of the strings below, action before description.

Reply:
Agreed, it would be nice to stay consistent.

Reply:
Yeah, I thought I was matching what err() etc did, but I was wrong. Fixed